tests/unit/lang/translator/de.txt

```diff
@@ -0,0 +1,162 @@
+Werkseinstellung
+Fehlerstat.
+Fil.-Sens.
+FILAMENT
+Filament
+Beende\ngepufferten gcode.
+Firmware Version\n
+Erste Schicht Kal.
+ERSTE SCHICHT KALIB.
+Flussfaktor
+FW UPDATE
+FW Update
+Heizbett
+Heizung wegen 30 Minuten Inaktivität deaktiviert.
+Hallo, hier ist Ihr\nOriginal Prusa MINI.
+HOME
+Home
+Im nächsten Schritt, \nden Knopf verwenden,\num die Höhe der Düse\neinzustellen. \nPrüfen Sie die\nBilder im Handbuch   \nals Referenz.
+INFO
+Info
+Führe ein
+IP-Adressen sind nicht gültig oder die Datei \lan_settings.ini\ befindet sich nicht im Stammverzeichnis des USB-Laufwerks.
+IP-Adressen oder Parameter sind nicht gültig oder die Datei \lan_settings.ini\ befindet sich nicht im Stammverzeichnis des USB-Laufwerks.
+Farbe korrekt?
+Filament im\nExtruder Antrieb?
+Ist Stahlblech auf dem Heizbett?
+LAN EINSTELLUNGEN
+Lan Einstellungen
+Live Adjust Z
+FILAMENT LADEN
+Filament laden
+Ladeeinstellungen
+Lade in Düse
+Laut
+Stellen Sie sicher,\ndass das Filament     \ndurch den Sensor  \ngeführt wird.
+Bettnivelierung.
+Bettnivellierung fehlgeschlagen?
+MELDUNGEN
+Meldungen
+Bewegen Achsen
+Bewegen E
+Bewegen X
+Bewegen Y
+Bewegen Z
+NEIN
+Kein Filamentsensor erkannt. Überprüfen Sie, ob der Sensor angeschlossen ist, und versuchen Sie es erneut.
+Kein USB
+Jetzt kalibrieren\nwir die Entfernung\nzwischen der Spitze\nder Düse und\ndas Druckblech.
+Düse
+Düse ist zu kalt.
+Aus
+An
+Bei Neustart
+Einmal
+Sobald der Drucker\nbeginnt, Kunststoff\nzu extrudieren,\nstellen Sie die Höhe\nder Düse durch Knopf\nDrehen ein, bis der\nFaden am Druckblech\nhaftet.
+Parke
+Legen Sie während der Kalibrierung der ersten 4 Punkte ein Blatt Papier unter die Düse. Wenn die Düse das Papier berührt, schalten Sie den Drucker sofort aus!
+Bitte reinigen Sie die Düse zur Kalibrierung. Klicken Sie auf WEITER, wenn Sie fertig sind.
+Bitte schließen Sie ein USB-Laufwerk an und versuchen Sie es erneut.
+Bitte schließen Sie\nden USB-Stick an,\nder mit Ihrem MINI \ngeliefert wurde und\nDrucker zurücksetzen\nzum Firmwareflashen
+Bitte legen Sie Stahlblech auf das Heizbett.
+Bitte entfernen Sie Stahlblech vom Heizbett.
+Bitte warten
+VORHEIZEN
+Vorheizen
+VORHEIZEN LADEN
+VORHEIZEN ENTLADEN
+Bereite Rammen vor
+WEITER drücken und\nFilament in den\nExtruder schieben.
+Drücken Sie WEITER,\num den Selbsttest\nauszuführen, bei dem\nnach potenziellen\nProblemen mit der\nMontage gesucht\nwird.
+Drucken
+Drucklüfter
+Druck wird enden
+Druckzeit
+FILAMENT REINIGEN
+Filament reinigen
+Reinige
+QR Test
+Ramme
+NEU HEIZEN
+Restzeit
+RESET DRUCKER
+WIEDERHOLEN
+Crash Dump speichern
+Einstellungen speichern
+Filamenttyp wählen
+Aktivieren, wenn Sie\nautomatisch aktua-\nlisierte Firmware\nvon einem USB-STICK\nflashen möchten.
+SelfTest
+SELBSTTEST
+Sende Info mit QR
+Service
+Einstellungen
+Einstellungen erfolgreich geladen
+Leise
+Ton Modus
+Ton-Typ
+Geschwindigkeit
+StandardAlert
+StandardPrompt
+Statische IPv4-Adressen wurden nicht festgelegt.
```

Comment:
IPv4-Adresse ... no "n" at the end if it refers to ONE IP ... if there are more possible then the "n" is correct.